doc/source/toolchain.rst

```diff
@@ -81,19 +81,20 @@ C Compilers
 ^^^^^^^^^^^
 
 SciPy is compatible with most modern C compilers (in particular ``clang``).
-However, CPython on Windows is
-built with specific versions of the Microsoft Visual C++ compiler [7]_, [8]_, [9]_,
-as is the corresponding build of SciPy. This has implications for the
-C language standards that can be supported [6]_.
+However, CPython on Windows is built with specific versions of the Microsoft
+Visual C++ compiler [7]_, [8]_, [9]_, as is the corresponding build of SciPy.
+This has implications for the C language standards that can be supported [6]_.
+Starting from MS Visual Studio 16.8, C11/C17 is supported [11]_ (without the
```

Comment:
This is great, and very much unexpected. Thanks for adding this!

Reply:
Happy to have contributed something :)